src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzer.cs

```diff
@@ -1509,43 +1499,62 @@ static void MergePlatformAttributes(ImmutableArray<AttributeData> immediateAttri
                                     }
                                 }
                             }
-                            // other platform attributes are ignored as the list couldn't be extended
+                            else // not existing parent platforms might need to be removed
```

Comment:
We generally prefer comments to be in a separate line inside the block. @CyrusNajmabadi another good analyzer rule for whitespace/newline related formatting style suggestions.